swagger.yaml

```diff
@@ -4630,3 +4630,35 @@ definitions:
         type: string
         enum: [beta, production]
         description: Level of support of the version.
+
+  operator-property:
+    type: object
+    properties:
+      name:
+        type: string
+        description: Name of the property
+      type:
+        type: string
+        enum: ['boolean', 'string', 'number']
+        description: Type of the property
+      options:
+        type: array
+        items:
+          type: string
+        description: Values to select from
+      description:
+        type: string
+        description: Description of a property
+      default_value:
+        type: string
+        description: Default value for the property
+
+  operator-properties:
+    type: object
+    properties:
+      operator_name:
+        $ref: '#/definitions/operator-type'
```

Comment:
why operator-type is needed in this case, it is user facing api, user should know only olm operators. There is no point to show him anything else

Reply:
In the master code operator-type was representing type of OLM operator (LSO, OCS, CNV); it needs to be changed to be in line with other changes.